2023.7/core/passed/qiime2-core-macos-latest-conda.yml

```diff
@@ -62,7 +63,7 @@ dependencies:
 - bioconductor-treesummarizedexperiment=2.6.0
 - bioconductor-xvector=0.38.0
 - bioconductor-zlibbioc=1.44.0
-- biom-format=2.1.12
+- biom-format=2.1.14
```

Comment:
@ebolyen, any reason `2.1.14` was used rather than `2.1.15.2` (released 5/8/23)? The changes in it are relatively minor

As a heads up, `2.1.16`, when I can get to it, includes significant optimization to rarefaction